tikv-client-store/src/lib.rs

```diff
@@ -1,30 +1,35 @@
 // Copyright 2018 TiKV Project Authors. Licensed under Apache-2.0.
+#![feature(type_alias_impl_trait)]
```

Comment:
Will this change stop users using this crate in stable channel?

Reply:
Yes, I've undone it in my next PR.